reformat.sh

```
@@ -1,7 +1,17 @@
 cd frontend
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)